### PR TITLE
Add update box86.sh pulled from TwisterOS 

### PR DESCRIPTION
**Pull request by mobilegmYT**
because running this is easier than manually cloning and recompiling.

### CODE REVIEW

**Comment by ptitSeb** [comment]
This is specific to TwisterOS, and box86 can be built on many other OS and plateform.
I can include that box86.sh, but not in the root forlder where people will want to use that.
Create some folder or use a name that indicate that at least.

File: update_box86.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd /home/pi/box86
```
Comment (rajdakin) [inline update_box86.sh:2]: Maybe use instead
```suggestion
cd "$(dirname "$0")"
```
so it is more portable? That would imply that there is no need to `cp` into `box86/backup` later, only to `backup`.

File: update_box86.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+sudo cp /usr/local/bin/box86 box86/backup
```
Comment (rajdakin) [inline update_box86.sh:3]: Why `sudo`? You can copy anything you have read access to -- which you do have here.

File: update_box86.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+cd box86/
+git pull origin master
+cd build
+make -j4
+sudo make install
+exit
+
+
```
Comment (rajdakin) [inline update_box86.sh:11]: This is just me being picky, but I hate files not ending with one single newline...

**Comment by mobilegmYT** [comment]
Ok. I moved the script into /src

**Comment by TuxThePenguin0** [comment]
If this script were to be included I feel like it'd have to be completely rewritten, nothing about it is even remotely portable given that it's essentially just a few lines that a regular user might enter into their terminal and it doesn't actually do much useful that the user couldn't do themselves by simply cloning the repo, followed by running the correct cmake and make commands for a given platform

**Comment by ptitSeb** [comment]
Yes, I don't plan to merge this PR as-is indeed. That's why it's still open for soo long...

**Comment by mobilegmYT** [comment]
Pale and Botspot have written their own script to install box86 easily and such, and this script is going to probably be included in there so I'll just close this PR for now.

**Pull request closed without merging**